secretmanager/api/SecretManagerSample/Program.cs

```diff
@@ -0,0 +1,381 @@
+/*
+ * Copyright (c) 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+using CommandLine;
+using Google.Cloud.Iam.V1;
+using Google.Cloud.Secrets.V1Beta1;
+using Google.Protobuf;
+using Google.Protobuf.WellKnownTypes;
+using Newtonsoft.Json.Linq;
+using Newtonsoft.Json;
+
+namespace GoogleCloudSamples
+{
+
+    class SecretOptions {
+        [Value(0, HelpText = "Full resource name of the secret", Required = true)]
+        public string name { get; set; }
```

Comment:
nit: This are better named with a capital N. If you are editing a file that already has some options named with first lowercase, then it's allright to keep to that, but since this is a new file, please, name all CamelCase.